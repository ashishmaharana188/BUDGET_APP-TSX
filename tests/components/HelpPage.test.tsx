import React from "react";
import { test, expect } from "vitest";
import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router-dom";
import HelpPage from "../../src/components/HelpPage";

test("should render HelpPage", () => {
  const { asFragment } = render(
    <MemoryRouter>
      <HelpPage />
    </MemoryRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});
