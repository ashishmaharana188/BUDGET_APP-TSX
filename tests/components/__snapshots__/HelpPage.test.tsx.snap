// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should render HelpPage 1`] = `
<DocumentFragment>
  <div>
    This is my help page.!
  </div>
</DocumentFragment>
`;
